packages/trader/src/Modules/SmartChart/Components/buy-toast-notification.jsx

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
```

Comment:
If you are creating a new file, it should be in typescript. Please change this to tsx

Reply:
sure